publisher_topic: motor_info_topic